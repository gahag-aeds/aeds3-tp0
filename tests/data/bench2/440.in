440 220
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 
Sub 276 421
Sum 240 385
Add 243 388
Sum 261 406
Add 257 402
Max 227 372
Add 165 310
Min 118 263
Sub 156 301
Min 229 374
Add 265 410
Sum 162 307
Add 123 268
Max 134 279
Sub 222 367
Max 59 204
Add 185 330
Sum 126 271
Add 247 392
Max 140 285
Sub 116 261
Min 64 209
Sub 50 195
Max 86 231
Sub 68 213
Sum 141 286
Sub 74 219
Max 146 291
Sub 90 235
Max 149 294
Sub 180 325
Sum 72 217
Sub 50 195
Max 128 273
Sub 288 433
Sum 12 157
Sub 16 161
Sum 159 304
Add 129 274
Sum 171 316
Add 49 194
Min 100 245
Add 289 434
Max 128 273
Add 189 334
Sum 216 361
Sub 142 287
Max 80 225
Add 3 148
Max 95 240
Add 175 320
Min 118 263
Sub 158 303
Sum 180 325
Add 203 348
Min 181 326
Sub 26 171
Min 232 377
Sub 282 427
Max 116 261
Sub 42 187
Max 167 312
Add 143 288
Max 92 237
Add 1 146
Min 136 281
Add 103 248
Max 266 411
Sub 250 395
Max 188 333
Add 99 244
Sum 255 400
Add 243 388
Sum 93 238
Sub 44 189
Max 137 282
Sub 264 409
Max 185 330
Sub 172 317
Max 266 411
Add 279 424
Max 8 153
Add 89 234
Max 98 243
Sub 188 333
Min 247 392
Sub 278 423
Sum 213 358
Sub 184 329
Max 266 411
Sub 284 429
Sum 225 370
Sub 138 283
Sum 132 277
Sub 272 417
Min 94 239
Sub 224 369
Min 37 182
Sub 22 167
Sum 135 280
Sub 224 369
Sum 120 265
Add 51 196
Max 128 273
Sub 168 313
Min 94 239
Sub 220 365
Max 137 282
Add 279 424
Sum 54 199
Sub 64 209
Sum 219 364
Add 61 206
Max 152 297
Add 23 168
Max 248 393
Sub 60 205
Sum 6 151
Add 167 312
Sum 243 388
Add 227 372
Sum 156 301
Add 129 274
Sum 27 172
Sub 244 389
Max 107 252
Sub 76 221
Sum 129 274
Add 143 288
Min 97 242
Add 263 408
Min 28 173
Sub 172 317
Max 20 165
Add 111 256
Sum 45 190
Add 69 214
Min 286 431
Add 137 282
Max 53 198
Add 45 190
Max 200 345
Sub 228 373
Min 106 251
Add 13 158
Min 250 395
Add 59 204
Sum 72 217
Add 211 356
Min 181 326
Sub 270 415
Sum 144 289
Add 293 438
Sum 105 250
Sub 126 271
Max 242 387
Add 167 312
Sum 201 346
Sub 76 221
Sum 15 160
Sub 4 149
Sum 294 439
Sub 292 437
Max 131 276
Add 269 414
Min 64 209
Sub 176 321
Max 44 189
Add 55 200
Sum 18 163
Sub 52 197
Min 100 245
Sub 218 363
Sum 279 424
Add 205 350
Max 230 375
Sub 234 379
Sum 219 364
Sub 258 403
Max 107 252
Sub 62 207
Max 233 378
Sub 206 351
Sum 60 205
Add 43 188
Min 37 182
Add 257 402
Max 209 354
Add 237 382
Sum 288 433
Add 179 324
Min 196 341
Add 287 432
Sum 132 277
Add 33 178
Min 262 407
Add 195 340
Max 164 309
Add 11 156
Min 250 395
Add 181 326
Max 62 207
Add 11 156
Sum 60 205
Add 3 148
Max 215 360
Sub 246 391
Sum 192 337
Add 95 240
Max 209 354
Sub 4 149
Sum 156 301
Add 147 292
Max 209 354
Add 171 316
Max 146 291
Add 245 390
Min 133 278
Sub 60 205
Sum 144 289
