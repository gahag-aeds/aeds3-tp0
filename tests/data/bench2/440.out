7163
7224
100
1
1
7732
100
100
7773
101
0
98
7514
100
100
6703
99
6519
7416
7638
-4
100
7743
95
96
-5
7685
-5
3
101
101
97
-7
102
101
7534
7093
102
102
101
100
99
1
7595
99
7382
7279
-6
-6
7219
7161
96
-6
94
6667
6798
94
96
6583
6829
7164
6569
96
7217
-5
-6
95
6601
-5
99
98
-1
-6
7354
-5
7569
7558
97
7158
6885
7085
100
-4
100
6726
-2
6827
97
6929
100
96
6815
-4
97
6855
-5
7520
-4
101
-3
101
7320
100
7532
101
7738
102
103
2
7974
